pkg/build/apiserver/registry/buildconfig/webhook.go

```diff
@@ -119,42 +123,50 @@ func (h *WebHookHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 func (w *WebHookHandler) ProcessWebHook(writer http.ResponseWriter, req *http.Request, ctx context.Context, name, subpath string) error {
 	parts := strings.Split(strings.TrimPrefix(subpath, "/"), "/")
 	if len(parts) != 2 {
+		glog.Error("")
```

Comment:
Don’t you want to dump all that state here like in the glog.Error call above @enj

Reply:
Was being quick and dirty since I just want to know at what call site we are failing.  Will enhance the debug at the exact failing point once I know.

Reply:
gotcha